src/pages/realiser-un-reportage/1.tsx

```diff
@@ -0,0 +1,129 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { TextField } from '@mui/material';
+
+import { getReportage } from 'src/activity-types/reportage.constants';
+import type { ReportageData } from 'src/activity-types/reportage.types';
+import { Base } from 'src/components/Base';
+import { StepsButton } from 'src/components/StepsButtons';
+import { Steps } from 'src/components/Steps';
+import { Activities } from 'src/components/activities/List';
+import { BackButton } from 'src/components/buttons/BackButton';
+import { ActivityContext } from 'src/contexts/activityContext';
+import { UserContext } from 'src/contexts/userContext';
+import { VillageContext } from 'src/contexts/villageContext';
+import { getQueryString } from 'src/utils';
+import { serializeToQueryUrl } from 'src/utils';
+import { ActivityStatus, ActivityType } from 'types/activity.type';
+import type { Activity } from 'types/activity.type';
+
+const ReportageStep1 = () => {
+  const router = useRouter();
+  const { axiosLoggedRequest } = React.useContext(UserContext);
+  const { village } = React.useContext(VillageContext);
+  const { activity, createNewActivity, updateActivity } = React.useContext(ActivityContext);
+  const [reportageActivity, setReportageActivity] = React.useState<Activity[]>([]);
```

Comment:
This variable name should be in plural if it is an array: `reportageActivities`.